src/main/webapp/readfile.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Read File Configuration</title>
+        <link rel="stylesheet" href="readfile.css">
+        <script src="/scripts/script.js"></script>
+    </head>
+    <body id="body">
+        <h1>Billing Configuration Sync Tool</h1>
+        </br>
+        <h2>Read Configuration</h2> 
+        <!-- Links Div -->
+        <div>
+        </div>
+        <!-- Search Div -->
+        <div>
+            <form>
+                
+                <p>Customer ID:</p>
+                <textarea></textarea>
+                </br>
+                <p>Account ID:</p>
+                <textarea></textarea>
+                </br>
+                <input type="submit">
+
+            </form>
+        </div>
+        </br>
+        <!-- Text Box Div -->
+        <div id="textbox">
+            <p>{
+              "legacy_customer_id": "STRNG",
+              "next_gen_customer_id": "INT",
+              "accounts":[
+                {
+                  "legacy_account_id": "STRING",
+                  "next_gen_customer_id": "INT",
+                  "settlement_attributes": {
+                    "currency_code": "STRING",
```

Comment:
Currently, you have the form in the JSON file format. I think you might just want to have each field listed on their own line. The JSON format may confuse the finance team. However, it is fine if this is the format you and Cade agreed on.

Reply:
It's more or less some placeholder text of how it should look like